models/secret/secret.go

```diff
@@ -93,3 +94,48 @@ func FindSecrets(ctx context.Context, opts FindSecretsOptions) ([]*Secret, error
 func CountSecrets(ctx context.Context, opts *FindSecretsOptions) (int64, error) {
 	return db.GetEngine(ctx).Where(opts.toConds()).Count(new(Secret))
 }
+
+// UpdateSecret changes org or user reop secret.
+func UpdateSecret(ctx context.Context, orgID, repoID int64, name, data string) error {
+	sc := new(Secret)
+	has, err := db.GetEngine(ctx).
+		Where("owner_id=?", orgID).
+		And("repo_id=?", repoID).
+		And("name=?", strings.ToUpper(name)).
```

Comment:
Here and below, the `ToUpper` doesn't seem consist with other code , eg: `InsertEncryptedSecret`

Reply:
See: https://github.com/go-gitea/gitea/blob/e4b2bdfbc0a28dcb4272de923b547a19131abd22/models/secret/secret.go#L34